web/src/components/snapshots/Snapshots.jsx

```diff
@@ -678,16 +678,27 @@ class Snapshots extends Component {
+            </div>
+            <div className="flex1 flex-column u-marginBottom--50 u-paddingLeft--20">
+              <p className="u-fontSize--small flex alignItems--center u-fontWeight--medium u-color--dustyGray u-marginTop--20"><span className="circleNumberGray u-marginRight--10"> 1 </span>Install the CLI on your machine by <a href="https://velero.io/docs/v1.3.2/basic-install/#install-the-cli" target="_blank" rel="noopener noreferrer" className="replicated-link u-marginLeft--5">following these instructions</a> </p>
+              <p className="u-fontSize--small flex alignItems--center u-fontWeight--medium u-color--dustyGray u-marginTop--20"><span className="circleNumberGray u-marginRight--10"> 2 </span> Install the Restic integration on your machince by <a href="https://velero.io/docs/v1.3.2/restic/" target="_blank" rel="noopener noreferrer" className="replicated-link u-marginLeft--5">following these instructions</a> </p>
+              <div className="flex flex1 u-marginTop--20">
+                <div className="flex">
+                  <span className="circleNumberGray u-marginRight--10"> 3 </span>
+                </div>
+                <div className="flex flex-column">
+                  <p className="u-fontSize--small flex alignItems--center u-fontWeight--medium u-color--dustyGray"> Run the commands from the instructions for your cloud provider </p>
+                  <div className="flex flex1 u-marginTop--15">
+                    <a href="https://github.com/vmware-tanzu/velero-plugin-for-aws#setup" target="_blank" rel="noopener noreferrer" className="snapshotOptions"> <span className="icon awsIcon u-cursor--pointer" /></a>
+                    <a href="https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure#setup" target="_blank" rel="noopener noreferrer" className="snapshotOptions"> <span className="icon azureIcon u-cursor--pointer" /></a>
+                    <a href="https://github.com/vmware-tanzu/velero-plugin-for-gcp#setup" target="_blank" rel="noopener noreferrer" className="snapshotOptions"> <span className="icon googleCloudIcon u-cursor--pointer" /></a>
+                    <a href="https://velero.io/docs/v1.3.2/supported-providers/" target="_blank" rel="noopener noreferrer" className="snapshotOptions"> <span className="icon cloudIcon u-cursor--pointer" /> Other </a>
+                  </div>
+                </div>
               </div>
+              <p className="u-fontSize--small flex alignItems--center u-fontWeight--medium u-color--dustyGray u-marginTop--20"><span className="circleNumberGray u-marginRight--10"> 4 </span> With all providers, you must install using the  <span className="inline-code u-marginLeft--5 u-marginRight--5"> --use-restic </span>  flag for snapshots to work. <a href="" target="_blank" rel="noopener noreferrer" className="replicated-link u-marginLeft--5">Learn more</a> </p>
```

Comment:
We should remove the "Learn more" link unless we have a doc or something that we can link to.